drivers/soundwire/intel.c

```diff
@@ -13,6 +13,7 @@
 #include <linux/io.h>
 #include <linux/pm_runtime.h>
 #include <sound/pcm_params.h>
+#include <linux/pm_runtime.h>
```

Comment:
duplicated <linux/pm_runtime.h>. It is at line 14

Reply:
yep, this comes from a bad rebase initially. thanks for spotting this.